docs/docs/features/preset-checks.md

```diff
@@ -41,7 +41,7 @@ To configure the preset checks, add the settings to the [recce config file](../r
 
 ### Recce Server
 
-1. When a new recce server is launched, all preset checks are automatically set up, but these checks are not executed at this time.
+1. When a new recce server instance is launched, all preset checks are automatically set up, but these checks are not executed at this time.
```

Comment:
what is "recce server instance"?

Reply:
Change all the "recce server instance" to "Recce instance"